tests/suite-integration/Entity.php

```diff
@@ -70,26 +67,27 @@ public function testDeleteEntity() {
       $agent = $this->newMockInstance(\PluginFlyvemdmAgent::class);
       $this->calling($agent)->notify->doesNothing;
       $agent->add([
-         'entities_id'        => $entityId,
-         '_email'             => $guestEmail,
-         '_invitation_token'  => $invitation->getField('invitation_token'),
-         '_serial'            => 'AZERTY',
-         'csr'                => '',
-         'firstname'          => 'John',
-         'lastname'           => 'Doe',
-         'version'            => '1.0.0',
+         'entities_id' => $entityId,
+         '_email' => $guestEmail,
+         '_invitation_token' => $invitation->getField('invitation_token'),
+         '_serial' => 'AZERTY',
+         'csr' => '',
+         'firstname' => 'John',
+         'lastname' => 'Doe',
+         'version' => '1.0.0',
```

Comment:
why not keeping key => val aligned ?

Reply:
Because is the auto formating of the editor, meaby to have les bits of spaces ;)